package.json

```diff
@@ -90,9 +90,11 @@
     "swiper": "^8.3.2",
     "tailwind-merge": "^1.3.0",
     "tailwindcss": "^3.1.4",
-    "yaml": "^2.1.3"
+    "yaml": "^2.1.3",
+    "@slack/web-api": "^6.5.1"
   },
   "devDependencies": {
+    "@netlify/functions": "^1.4.0",
```

Comment:
Why we add `@netlify/functions` as a devDependency as we need netlify cli to integrate Netlify functions with frontend?

Reply:
I have only been using some types from `@netlify/functions` and it's cli for testing functions. that's why I think it should be in devDependencies.